src/media/js/site/components/header.js

```diff
@@ -8,30 +8,56 @@ export default class Header extends React.Component {
   static contextTypes = {
     router: React.PropTypes.object
   };
-  renderMenu() {
+
+  stubMenuItem(message) {
```

Comment:
don't think we need to add this stuff if we're going to strip it later. like adding a bow tie to the skeleton when we know we're gonna put on a tie